### ISSUE TITLE
Resolve DNS Issues with Azure

### ISSUE THREAD

**Issue by arthomasredhat** [redhat-manufacturing/osdu-lab-gitops#6]

**Comment by arthomasredhat** [comment]
@tannermoore Do you have an update on this task?

**Comment by arthomasredhat** [comment]
Unable to get feedback from Paul from Chevron.  We need a service principle. Christopher@chevron.com

**Comment by codekow** [comment]
FYI... https://cert-manager.io/docs/configuration/acme/dns01/azuredns/#service-principal

**Comment by arthomasredhat** [comment]
We need to deploy our own Azure AD tenant. This will ultimately remove the obstacle have this deployed.

**Comment by codekow** [comment]
@arthomasredhat I have a solution for this in route53. We can create subdomains and delegate to other groups.

Sync up with me.

**Comment by codekow** [comment]
I'm planning to redeploy OCP with a new domain name `*.apps.edgelab.online`. Ping me if you have questions. DNS is setup in route53 and we can delegate subdomains to other groups.

**Comment by tannermoore** [comment]
I heard back from Christopher and he's ok to create an SP for us. Let me know if we still have a need for one in their tenant after deployment.

**Comment by arthomasredhat** [comment]
Yes, we need the tenant after the deployment.  Per @codekow, we have route 53 set up.

**Comment by codekow** [comment]
FYI... new OCP console is here https://console-openshift-console.apps.hou.edgelab.online/

**Comment by jarcher** [comment]
Need Access to Route 53 and discuss route creation automation

**Comment by codekow** [comment]
My plan is to create subdomains and let individual groups manage those subdomains however they want. Please help me understand your use case so I can help setup a solution.
